terraform/modules/monitoring/prober.tf

```diff
@@ -34,6 +57,10 @@ resource "google_cloud_run_v2_job" "pmap_prober" {
 
   location = "us-central1"
 
+  depends_on = [
+    google_project_service.prober_services["cloudscheduler.googleapis.com"],
```

Comment:
actually, it is better to add this dependency in resource "google_cloud_scheduler_job" "job"?

Reply:
cloud scheduler job depends on `google_cloud_run_v2_job.pmap_prober`

Reply:
Thanks for catching it. This is a typo.